pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -609,10 +613,22 @@ func handleInitVerifiedObjects(ctx *verifierContext) {
 func verifyImageStatusFromImageFile(imageFileName string,
 	size int64, pathname string) *types.VerifyImageStatus {
 
+	// filename might have two parts, separated by '.': digest and base64(mediaType)
```

Comment:
base64 no longer correct.
Also please add something saying that the media type can be missing after an upgrade from previous versions of EVE-OS.

Reply:
> Also please add something saying that the media type can be missing after an upgrade from previous versions of EVE-OS

I can see a scenario where it happens during any reboot, not just upgrade. The timing has to be just right (or wrong), though.

Comment added.